src/components/Indicator/index.jsx

```diff
@@ -2,41 +2,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import cc from 'classcat';
 
-export const VALID_TYPES = ['update', 'count'];
+export const SIZES = ['xs', 's'];
```

Comment:
This could be cleaner as a `git mv` - I'm assuming there aren't actual code changes being made with this branch...

Reply:
discussed on zoom, going to just merge in as is for now but I'll try to avoid this in the future.